controllers/policy_utils.go

```diff
@@ -193,6 +200,11 @@ func reconcilePolicyDeletion(ctx context.Context, client client.Client, policy p
 			return ctrl.Result{}, err
 		}
 	}
+	// Remove the old finalizer used to ensure that the policy server created
+	// before this controller version is delete as well. As the upgrade path
+	// supported by the Kubewarden project does not allow jumping versions, we
+	// can safely remove this line of code after a few releases.
+	controllerutil.RemoveFinalizer(policy, "kubewarden")
```

Comment:
Maybe worth to add a constant for it, such as `constants.KubewardenFinalizerPre113` or similar.

Reply:
Yes, having a constant would be nice if you don't mind